privateSBOMExchange/evaluation/intellectual_property_policy

```diff
@@ -0,0 +1,16 @@
+# This policy defines access control over sbom data to ensure only authorized users can view sensitive information.
+# The policy maker's intention is to restrict the access to information that might cause intellectual property leakage of the software artifact that the SBOM describes.
+# To achive this, the policy restricts access to all to all dependencies information.
+# Atribute types/catagories are: User Role, department , Audit Authorization.
+
+# Policy Definition
+# ..................
+# Personnel/organizations/ or tool who can view the above data must hold one of the following attribute combinations:
+# Rule 1 : Compliance Auditor can only 
+# Rule 2 : The following are granted full access to all dependency information:
+    # Intellectual Property Owners.
+    # Security Auditor with audit authorization status of approved.
+    # Authorized personnel with a role of Security Analyst, who works in the DoD.
+[package]
+name = "compliance auditor"
+* = "IP owner" or ("Security Auditor" and "Audit Authorization status of Approved") or ("Security Analyst" and "works at DoD" ) 
```

Comment:
I'm assuming this is the actual policy that will be ingested by Petra? Can you please add a 1-sentence explanation for how Petra will interpret this policy?